lib/gamera/page.rb

```diff
@@ -136,7 +136,7 @@ class Page
 
     def initialize(url, url_matcher = nil)
       @url = url
-      @url_matcher = url_matcher
+      @url_matcher = url_matcher || /#{url}/
```

Comment:
probably want `/#{Regexp.escape(url)}/`. and might want `/^#{Regexp.escape(url)}/` or `/^#{Regexp.escape(url)}$/` or `/#{Regexp.escape(url)}$/`.  You'll know more use cases than I do for whether the default matcher should be locked to the start or end or both of the url.